Scripts/Python/step_response_identification.py

```diff
@@ -0,0 +1,122 @@
+import pandas as pd
```

Comment:
The path for your script must be Scripts/Python/SISO. This SISO folder is important for future distinction from MIMO scripts.